activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -479,6 +479,7 @@ def enable_extension(name, **)
       #   Set to +:cascade+ to drop dependent objects as well.
       #   Defaults to false.
       def disable_extension(name, force: false)
+        _schema, name = name.to_s.split(".").values_at(-2, -1)
```

Comment:
Not sure I understood this, but you want the last 2 values?
```suggestion
        *, _schema, name = name.to_s.split(".")
```

Reply:
This was taken from `enable_extension`, which splits the `name` the same way. No issue with your proposed change, but then we should probably change both places (or perhaps extract the logic to a separate method?)

Reply:
I'd say we should change this if it's more efficient than the previous approach. Otherwise leave it as is.

Reply:
Not likely to be more/less efficient either way, so let's leave as-is assuming you're comfortable with the minor duplication of the parsing logic (i.e. this one line) in `enable_extension` and `disable_extension`.